lib/Dialect/FIRRTL/Transforms/LowerMemory.cpp

```diff
@@ -252,9 +252,9 @@ void LowerMemoryPass::lowerMemory(MemOp mem, const FirMemory &summary,
   auto memModule = getOrCreateMemModule(mem, summary, ports, shouldDedup);
   b.setInsertionPointToStart(wrapper.getBodyBlock());
 
-  auto memInst =
-      b.create<InstanceOp>(mem->getLoc(), memModule, memModule.getModuleName(),
-                           mem.getNameKind(), mem.getAnnotations().getValue());
+  auto memInst = b.create<InstanceOp>(
+      mem->getLoc(), memModule, (mem.getName() + "_ext").str(),
```

Comment:
Aside: It's annoying that this needs a string ref and can't be done with a Twine.

Reply:
Two things:

1. Would it be better to just use the original memory name here, i.e., `mem.getName()`?
2. If generating a new name, this can conflict with names in the module namespace and would need to go through a `newName` API to generate a name. (Will this error if I have a wire called `mem_ext` and a memory called `mem`?)

Reply:
For (2), this actually doesn't matter as the name will get uniqued. This is only a problem for symbols.

Reply:
> Would it be better to just use the original memory name here, i.e., mem.getName()?

I think that would be better, but I advised @youngar to keep it _for now_ because I know users rely on `_ext` in brittle regex flows.